src/mono/wasm/build/WasmApp.targets

```diff
@@ -362,11 +368,18 @@
 
   </Target>
 
+  <Target Name="_EnsureWasmRuntimeAssetsLocation" Condition="'$(WasmRuntimeAssetsLocation)' == ''">
+    <Message Importance="Low" Text="Set 'WasmRuntimeAssetsLocation' to default value '_framework'" />
+    <PropertyGroup>
+      <WasmRuntimeAssetsLocation>_framework</WasmRuntimeAssetsLocation>
+    </PropertyGroup>
+  </Target>
```

Comment:
Based on @pavelsavara feedback, I added the message to inform the user that the location is overriden to default value (`_framework`), mostly because setting the property (by user) to empty value has no effect.

Reply:
The user isn't really going to see the message since it's emitted with importance=low. This defaulting to `_framework` sounds like something to be in documentation.

Reply:
Yes, it's meant to be in the binlog. The behavior is described at the top of the `WasmApp.targets`.

Reply:
I don't see the value of having this special message for this one property. We don't do this for any other public property with default values.
Maybe instead of this you could emit an explicit message in WasmAppBuilder that the particular path is being used as the location for the runtime assets.

Reply:
Ook